mopidy/core/library.py

```diff
@@ -228,8 +228,9 @@ def lookup(self, uri=None, uris=None):
 
         # TODO: lookup(uris) to backend APIs
         for backend, backend_uris in self._get_backends_to_uris(uris).items():
-            for u in backend_uris:
-                futures[(backend, u)] = backend.library.lookup(u)
+            if backend_uris:
```

Comment:
Could also be `for u in backend_uris or []` I think, but either way is fine with me.